website/common/script/i18n.js

```diff
@@ -60,4 +61,26 @@ function t (stringName) {
   }
 }
 
+// translates passed strings to two (passed) languages and concatenates them
+// returns an array
+function translateTwice (languages, messages) {
```

Comment:
this function needs tests, but to be honest I think it would be better to just call `i18n.t` twice, with same parameters except for the language. I find this function to be a little confusing and not really necessary